.docs/user-guide/storage-providers.md

```diff
@@ -707,3 +707,107 @@ libstorage:
   in place, it may be possible for a client to attach a volume that is already
   attached to another node. Mounting and writing to such a volume could lead to
   data corruption.
+
+## GCEPD
+
+The Google Compute Engine (GCE) Persistent Disk driver registers a driver named
```

Comment:
I left this one on purpose. But not for a strong reason. I can change it as well.